scripts/globals/dealer_moogle.lua

```diff
@@ -0,0 +1,545 @@
+    [30] =
+    {
+
+    },
+
+    -- Kupon W-DeIII: Delve boss weapons
+    [31] =
+    {
+
+    },
+
+    -- Kupon AW-Mis: Equipment or a weapon from High-Tier Mission Battlefields
+    [32] =
+    {
+
+    },
+
+    -- Kupon AW-Vgr: Equipment from Vagary Notorious Monsters Perfiden and Plouton
+    [33] =
+    {
+
+    },
+
+    -- Kupon AW-VgrII: Equipment from Vagary NMs Palloritus, Putraxia and Rancibus
+    [34] =
+    {
+
+    },
+
+    -- Mog Kupon W-Pulse: Pulse weapons
+    [35] =
+    {
+        xi.items.GIRRU,
+    },
+
+    -- Kupon AW-GFII: Geas Fete (Content Level 119+) (deprecated)
+    [37] =
+    {
+
+    },
+
+    -- Kupon I-Stone: Augment stones (deprecated)
+    [38] =
+    {
+
+    },
+
+    -- Kupon AW-GF: Geas Fete (Content Level 119) (deprecated)
+    [39] =
+    {
+
+    },
+
+    -- Kupon AW-UWII: Level 119-128 Unity Wanted Battles
+    [40] =
+    {
+
+    },
+
+    -- Kupon AW-UW: Level 119 Unity Wanted Battles
+    [41] =
+    {
+
+    },
+
+    -- Kupon A-Ab: iLevel 119 +1 Abjuration Armor pieces
+    [43] =
+    {
+
+    },
+
+    -- Kupon AW-Cos: Various Costumes
+    [44] =
+    {
+
+    },
+
+    -- Kupon AW-Kupo: All four Kupo items
+    [45] =
+    {
+        {
+            -- TODO: Move to items.lua
+            21074, -- Kupo Rod
+            25645, -- Kupo Masque
+            25726, -- Kupo Suit
+            26406, -- Kupo Shield
+        },
+    },
+
+    -- Kupon W-EMI: iLevel 117 Sparks of Eminence Weapons
+    [46] =
+    {
+
+    },
+
+    -- Kupon A-EMI: iLevel 117 Records of Eminence armor
+    [47] =
+    {
+
+    },
+
+    -- Kupon W-SRW: Rala Waterways Skirmish Weapons +2
+    [48] =
+    {
+
+    },
+
+    -- Kupon W-SCC: Cirdas Caverns Skirmish Weapons +2
+    [49] =
+    {
+
+    },
+
+    -- Kupon A-SYW: Yorcia Weald Skirmish Armor +1
+    [50] =
+    {
+
+    },
+
+    -- Kupon W-ASRW: Rala Waterways Alluvion Skirmish Weapons
+    [51] =
+    {
+
+    },
+
+    -- Kupon W-ASCC: Cirdas Caverns Alluvion Skirmish Weapons
+    [52] =
+    {
+
+    },
+
+    -- Kupon A-ASYW: Yorcia Weald Alluvion Skirmish Armor
+    [53] =
+    {
+
+    },
+
+    -- Kupon W-R119: iLevel 119 III Relic Weapons (deprecated)
+    [54] =
+    {
+
+    },
+
+    -- Kupon W-M119: iLevel 119 III Mythic Weapons and Ergon Weapons (deprecated)
+    [55] =
+    {
+
+    },
+
+    -- Kupon W-E119: iLevel 119 III Empyrean Weapons (deprecated)
+    [56] =
+    {
+
+    },
+
+    -- Kupon W-A119: Aeonic Weapons (deprecated)
+    [57] =
+    {
+
+    },
+
+    -- Kupon A-OmII: Body pieces from Omen bosses
+    [59] =
+    {
+
+    },
+
+    -- Kupon I-AF119: Scale needed for the Reforged Artifact Armor +3 process
+    [60] =
+    {
+
+    },
+
+    -- Kupon AW-Om: Equipment pieces from Omen mid-bosses
+    [61] =
+    {
+
+    },
+
+    -- Kupon W-RMEA: iLevel 119 III Relic, Mythic, Empyrean or Aeonic Weapon
+    [62] =
+    {
+
+    },
+    ]]
+}
+
+-- WARNING: These items cannot be customised!
```

Comment:
Just a styling thing, but would it be possible to move this table above the item list?  For someone reading top to bottom to understand what the keys in that table are for (and not just linear).  Either that, or a related comment above similar to here where these values are client-defined